bin/spiced/src/lib.rs

```diff
@@ -42,6 +42,8 @@ pub struct Args {
 
 pub async fn run(args: Args) -> Result<()> {
     let app = App::new(".").context(UnableToConstructSpiceAppSnafu)?;
+    let auth = runtime::auth::get();
```

Comment:
This is fine for now when Spice is the only provider - but we will need a way to load datasource specific auth based on which datasources are configured.